<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Escorpio - LogBug</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/assets.qrc">
    <normaloff>:/LogBug.png</normaloff>:/LogBug.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Log level:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="infoCheck">
              <property name="text">
               <string>Information</string>
              </property>
              <property name="icon">
               <iconset resource="assets/assets.qrc">
                <normaloff>:/info.png</normaloff>:/info.png</iconset>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="warningCheck">
              <property name="text">
               <string>Warning</string>
              </property>
              <property name="icon">
               <iconset resource="assets/assets.qrc">
                <normaloff>:/warning.png</normaloff>:/warning.png</iconset>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="errorCheck">
              <property name="text">
               <string>Error</string>
              </property>
              <property name="icon">
               <iconset resource="assets/assets.qrc">
                <normaloff>:/error.png</normaloff>:/error.png</iconset>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Tag filter:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tagFilterText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="logTable">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="okButton">
               <property name="text">
                <string>OK</string>
               </property>
               <property name="icon">
                <iconset resource="assets/assets.qrc">
                 <normaloff>:/save.png</normaloff>:/save.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancelButton">
               <property name="text">
                <string>Cancel</string>
               </property>
               <property name="icon">
                <iconset resource="assets/assets.qrc">
                 <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QGroupBox" name="parametersBox">
               <property name="title">
                <string>Serial port</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="2" column="1">
                 <widget class="QComboBox" name="parityBox"/>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="flowControlLabel">
                  <property name="text">
                   <string>Flow control:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="stopBitsLabel">
                  <property name="text">
                   <string>Stop bits:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="stopBitsBox"/>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="flowControlBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="dataBitsLabel">
                  <property name="text">
                   <string>Data bits:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="dataBitsBox"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="parityLabel">
                  <property name="text">
                   <string>Parity:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="baudRateLabel">
                  <property name="text">
                   <string>BaudRate:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="baudRateBox"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReload"/>
   <addaction name="actionPause"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionScroll"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/export.png</normaloff>:/export.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/refresh.png</normaloff>:/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/record.png</normaloff>:/record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionScroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/down.png</normaloff>:/down.png</iconset>
   </property>
   <property name="text">
    <string>Scroll</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="assets/assets.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
